inlong-manager/manager-dao/src/main/resources/mappers/AuditEntityMapper.xml

```diff
@@ -38,16 +38,20 @@
     <resultMap id="SumByLogTsResultMap" type="java.util.Map">
         <result column="log_ts" property="logTs" jdbcType="VARCHAR"/>
         <result column="total" property="total" jdbcType="BIGINT"/>
+        <result column="total_delay" property="totalDelay" jdbcType="BIGINT"/>
     </resultMap>
 
     <select id="sumByLogTs" resultMap="SumByLogTsResultMap">
         select date_format(log_ts, #{format, jdbcType=VARCHAR}) as log_ts, sum(`count`) as total, sum(`delay`) as total_delay
-        from apache_inlong_audit.audit_data
-        where inlong_group_id = #{groupId,jdbcType=VARCHAR}
-          and inlong_stream_id = #{streamId,jdbcType=VARCHAR}
-          and audit_id = #{auditId,jdbcType=VARCHAR}
-          and log_ts &gt;= #{sDate, jdbcType=VARCHAR}
-          and log_ts &lt; #{eDate, jdbcType=VARCHAR}
+        from (
+            select distinct ip, docker_id, thread_id, sdk_ts, packet_id, log_ts, inlong_group_id, inlong_stream_id, audit_id, `count`, `size`, `delay`
```

Comment:
Do we really need query so much columns here?

Reply:
All audit fields are included here because the requirement is to duplicate audit data according to all audit fields.